deployment/GI03FLMB_Deploy.csv

```diff
@@ -103,55 +103,55 @@ AR21,,AR30-03,,GI03FLMB-RIS01-00-SIOENG000,4,1,2017-08-07T15:49:00,2018-06-11T19
-AR30-03,,,,GI03FLMB-RIS01-04-PHSENF000,5,1,2018-06-11T14:35:00,,CGMGI-03FLMB-00005,,CGINS-PHSENF-P0181,59.71427,-39.31329,,30,2829,
-AR30-03,,,,GI03FLMB-RIS01-05-FLORTD000,5,1,2018-06-11T14:35:00,,CGMGI-03FLMB-00005,,CGINS-FLORTD-01320,59.71427,-39.31329,,30,2829,
-AR35-05,,,,GI03FLMB-RI000-00-CTDMOH000,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13454,,,,1830,2812,
-AR35-05,,,,GI03FLMB-RI000-00-CTDMOH100,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13457,,,,2730,2812,
-AR35-05,,,,GI03FLMB-RI000-00-CTDMOH400,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13456,,,,2430,2812,
-AR35-05,,,,GI03FLMB-RI000-00-CTDMOH700,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13455,,,,2130,2812,
-AR35-05,,,,GI03FLMB-RI000-00-VELPTB000,6,1,,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12236,,,,1830,2812,
-AR35-05,,,,GI03FLMB-RI000-00-VELPTB100,6,1,,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12673,,,,2730,2812,
-AR35-05,,,,GI03FLMB-RI000-00-VELPTB400,6,1,,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12663,,,,2430,2812,
-AR35-05,,,,GI03FLMB-RI000-00-VELPTB700,6,1,,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12459,,,,2130,2812,
-AR35-05,,,,GI03FLMB-RIM01-00-SIOENG000,6,1,,,CGMGI-03FLMB-00006,,CGCON-SIOCON-14874,,,,1507,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-ADCPSL007,6,1,,,CGMGI-03FLMB-00006,,CGINS-ADCPSL-23381,,,,500,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG060,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11594,,,,30,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG061,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-12609,,,,40,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG062,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11645,,,,60,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG063,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11604,,,,90,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG064,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11534,,,,130,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG065,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11606,,,,180,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG066,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11608,,,,250,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG067,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11603,,,,350,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG068,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-12190,,,,500,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH069,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-12227,,,,750,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH070,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-11703,,,,1000,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH071,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-12219,,,,1500,2812,
-AR35-05,,,,GI03FLMB-RIS01-00-SIOENG000,6,1,,,CGMGI-03FLMB-00006,,CGCON-SIOCON-12372,,,,30,2812,
-AR35-05,,,,GI03FLMB-RIS01-03-DOSTAD000,6,1,,,CGMGI-03FLMB-00006,,CGINS-DOSTAD-00224,,,,30,2812,
-AR35-05,,,,GI03FLMB-RIS01-04-PHSENF000,6,1,,,CGMGI-03FLMB-00006,,CGINS-PHSENF-P0183,,,,30,2812,
-AR35-05,,,,GI03FLMB-RIS01-05-FLORTD000,6,1,,,CGMGI-03FLMB-00006,,CGINS-FLORTD-01221,,,,30,2812,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH000,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-10266,59.71427,-39.31329,,1830,2829,

```

Comment:
All of the OSNAP instruments should be xx29 m

Reply:
This is what you sent via email:
Please note that the deployed depths of the OSNAP instruments on the 2 Flanking Moorings will need to be updated:
Off Bottom 	FLMA-6   	FLMB-6
   1000 m 	1692 m     	1819 m 
    700 m   	1992 m    	2119 m 
    400 m   	2292 m    	2419 m 
    100 m   	2592 m    	2719 m

Reply:
My note was for FLMB-6.  This is FLMB-5 which was just recovered.  If the deployment depth is 2829 m, then 1000 m off the bottom is 1829 m, not 1830 m which what is listed.

Reply:
ok then.  Fixing.